src/scripts/quick_tags.js

```diff
@@ -159,15 +160,20 @@ const addTagsToPost = async function ({ postElement, inputTags = [] }) {
   tags.push(...tagsToAdd);
 
   try {
-    const { response: { displayText } } = await apiFetch(`/v2/blog/${uuid}/posts/${postId}`, {
-      method: 'PUT',
-      body: {
-        ...createEditRequestBody(postData),
-        tags: tags.join(',')
-      }
-    });
-
-    notify(displayText);
+    if (isNpfCompatible(postData)) {
+      const { response: { displayText } } = await apiFetch(`/v2/blog/${uuid}/posts/${postId}`, {
+        method: 'PUT',
+        body: {
+          ...createEditRequestBody(postData),
+          tags: tags.join(',')
+        }
+      });
+
+      notify(displayText);
+    } else {
+      await megaEdit([postId], { mode: 'add', tags: tagsToAdd });
+      notify(`Edited on ${blogName} (legacy)`);
```

Comment:
could make this copy more accurate

```suggestion
      notify(`Edited legacy post on ${blogName}`);
```